src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerFullGroupTable.class.php

```diff
@@ -441,6 +433,51 @@ public function AddFields()
         $this->AddCustomColumns();
     }
 
+    /**
+     * @param array $cellConfig
+     *
+     * @return null|array|string - returns the types AddColumn allows for the callback parameter: null, array('object','callbackName'), callbackNameString
+     */
+    protected function getCellFormattingFunctionConfig(array $cellConfig)
+    {
+        $formatFunctionName = $cellConfig['callback_fnc'];
+
+        if ('' === $formatFunctionName) {
+            $cellFormattingConfig = $this->getGlobalCellFormattingFunctionConfigByCellConfig($cellConfig);
+
+            if (null === $cellFormattingConfig) {
+                return $cellFormattingConfig;
+            }
+        }
+
+        if ('gcf_' === substr($formatFunctionName, 0, 4) || 'ccf_' === substr($formatFunctionName, 0, 4)) {
+            return $formatFunctionName;
+        }
+
+        return $this->getCellFormattingFunctionConfigByRecordObject($formatFunctionName);
+    }
+
+    protected function getGlobalCellFormattingFunctionConfigByCellConfig(array $cellConfig): ?array
+    {
+        if ('id' === $cellConfig['db_alias'] && 'ID' === $cellConfig['title']) {
+            return array('TCMSRecord', 'callBackUuid');
```

Comment:
Could/should this be moved to TCMSRecord?

(because also this is "by record object" - in this case all objects)

Reply:
combined the methods and moved it to the TCMSRecord base class.